nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-edit-layout.html

```diff
@@ -72,7 +71,6 @@
       directory-name="l10ncoverage"
       name="coverage"
       value="{{document.properties.dc:coverage}}"
-      dbl10n="true"
```

Comment:
These layouts are useless but `dbl10n="true"` must be kept since coverage is such vocabulary